Core/src/org/sleuthkit/autopsy/corecomponents/DataContentViewerHTML.java

```diff
@@ -0,0 +1,293 @@
+/*
+ * Autopsy Forensic Browser
+ *
+ * Copyright 2011-2016 Basis Technology Corp.
+ * Contact: carrier <at> sleuthkit <dot> org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.sleuthkit.autopsy.corecomponents;
+
+import org.sleuthkit.autopsy.modules.filetypeid.FileTypeDetector;
+import java.awt.BorderLayout;
+import java.awt.Component;
+import java.util.logging.Level;
+import com.sun.javafx.application.PlatformImpl;
```

Comment:
This is a internal API and subject to change.  You should probably be using javafx.application.Platform